cypress/e2e/alerts.cy.js

```diff
@@ -1,25 +1,47 @@
 describe('Cypress application', () => {
   before(() => {
```

Comment:
Site page with alerts only opens before first test. For another test site doesn't open. So please make changes